cmd/influx/auth.go

```diff
@@ -65,86 +70,40 @@ func newCreateCommand() cli.Command {
 			Usage: "The bucket id",
 		},
 		&cli.BoolFlag{
-			Name:        "write-tasks",
-			Usage:       "Grants the permission to create tasks",
-			Destination: &params.WriteTasksPermission,
-		},
-		&cli.BoolFlag{
-			Name:        "read-tasks",
-			Usage:       "Grants the permission to read tasks",
-			Destination: &params.ReadTasksPermission,
-		},
-		&cli.BoolFlag{
-			Name:        "write-telegrafs",
-			Usage:       "Grants the permission to create telegraf configs",
-			Destination: &params.WriteTelegrafsPermission,
-		},
-		&cli.BoolFlag{
-			Name:        "read-telegrafs",
-			Usage:       "Grants the permission to read telegraf configs",
-			Destination: &params.ReadTelegrafsPermission,
-		},
-		&cli.BoolFlag{
-			Name:        "write-orgs",
-			Usage:       "Grants the permission to create organizations",
-			Destination: &params.WriteOrganizationsPermission,
-		},
-		&cli.BoolFlag{
-			Name:        "read-orgs",
-			Usage:       "Grants the permission to read organizations",
-			Destination: &params.ReadOrganizationsPermission,
-		},
-		&cli.BoolFlag{
-			Name:        "write-dashboards",
-			Usage:       "Grants the permission to create dashboards",
-			Destination: &params.WriteDashboardsPermission,
-		},
-		&cli.BoolFlag{
-			Name:        "read-dashboards",
-			Usage:       "Grants the permission to read dashboards",
-			Destination: &params.ReadDashboardsPermission,
-		},
-		&cli.BoolFlag{
-			Name:        "write-checks",
-			Usage:       "Grants the permission to create checks",
-			Destination: &params.WriteCheckPermission,
-		},
-		&cli.BoolFlag{
-			Name:        "read-checks",
-			Usage:       "Grants the permission to read checks",
-			Destination: &params.ReadCheckPermission,
-		},
-		&cli.BoolFlag{
-			Name:        "write-notificationRules",
-			Usage:       "Grants the permission to create notificationRules",
-			Destination: &params.WriteNotificationRulePermission,
+			Name:        "operator",
```

Comment:
Instead of having `--operator` and `--all-access` flags, what do you think of making them be separate subcommands (i.e. `auth create --all-access`)? I ask because it doesn't really make sense to do something like `auth create --all-access --write-buckets abcd`

Reply:
I see them as groups of permissions - in theory we could have another grouping of `--dashboard-read` or something. And composing groups of permissions with extra individual ones makes sense to me.

Reply:
And something like `--all-access --read-orgs` might make sense even now.

Reply:
@russorat FYI

Reply:
In the end I updated this so `--operator` and `--all-access` are still flags, but it now gives an error if you try to use either of these together with finer-grained permission flags.